prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+  #card-deck {
+    display: flex;
+  }
+  div {
+    flex-wrap: wrap;
+    flex-basis:calc(100% / 6)
+  }
```

Comment:
Hi Jorge, for all of your files here, be sure to put `flex-basis` on the `.card-wrapper` instead of all `div` elements. 

`flex-basis` is meant to be used on the children of elements with `display: flex`